Modules/Scripted/SampleData/SampleData.py

```diff
@@ -232,14 +224,16 @@ def __str__(self):
             "customDownloader  : %s" % self.customDownloader,
             "",
         ]
-        for fileName, uri, nodeName, loadFile, fileType, checksum in zip(self.fileNames, self.uris, self.nodeNames, self.loadFiles, self.loadFileType, self.checksums):
+        for fileName, uri, nodeName, loadFile, fileType, checksum in zip(
+            self.fileNames, self.uris, self.nodeNames, self.loadFiles, self.loadFileTypes, self.checksums):
+
             output.extend([
-                " fileName    : %s" % fileName,
-                " uri         : %s" % uri,
-                " checksum    : %s" % checksum,
-                " nodeName    : %s" % nodeName,
-                " loadFile    : %s" % loadFile,
-                " loadFileType: %s" % fileType,
+                " fileName     : %s" % fileName,
```

Comment:
I suggest to add a first commit titled `STYLE: Improve SampleData source code readability` where all stylistic changes would be done, we will then add to it the `.git-blame-ignore-revs`[^1] file once the MR is merged.

[^1]: https://github.com/Slicer/Slicer/blob/main/.git-blame-ignore-revs